package.json

```diff
@@ -16,17 +16,18 @@
     "lint": "turbo lint",
     "prettier": "turbo run prettier",
     "test": "turbo run test",
-    "prepare": "husky install"
+    "prepare": "husky"
```

Comment:
`husky install` is deprecated, and just `husky` is the current preferred command.